appintro/src/main/java/com/github/appintro/model/SliderPage.kt

```diff
@@ -25,6 +26,7 @@ data class SliderPage @JvmOverloads constructor(
     var title: CharSequence? = null,
     var description: CharSequence? = null,
     @DrawableRes var imageDrawable: Int = 0,
+    var startAnimation: Boolean = false,
```

Comment:
I think we should not add `startAnimation` as a boolean flag to `SliderPage` (and generally be exposed).

Reply:
Agree, its not required.